.gitignore

```diff
@@ -1,3 +1,7 @@
 dist/
 node_modules/
 target/
+/test-results/
+/playwright-report/
+/blob-report/
```

Comment:
?

Reply:
gonna reduce it ... I think only playwright-report is actually used ... auto-generated stuff :blush: